docs/api/app.md

```diff
@@ -348,6 +348,25 @@ bar, and on OS X you can visit it from dock menu.
 
 Clears the recent documents list.
 
+### `app.setAsDefaultProtocolClient(protocol)` _OS X_ _Windows_
```

Comment:
I'm mostly thinking about the uninstall case - on Mac people will delete the app and launchd will react appropriately, but on Windows you leave a trash regkey around